libbeat/autodiscover/providers/kubernetes/pod.go

```diff
@@ -435,14 +437,16 @@ func (p *pod) podEvent(flag string, pod *kubernetes.Pod, ports common.MapStr, in
 	if len(namespaceAnnotations) != 0 {
 		kubemeta["namespace_annotations"] = namespaceAnnotations
 	}
+	orchestrator := genOrchestratorFields(kubemeta, "pod")
```

Comment:
Is there some guidance around the `orchestrator.resource.type` field? Just making sure it makes sense to put `pod` here, the examples in the docs all have `service` and not sure if there was a desire to make the language orchestrator agnostic or not.

Reply:
Kubernetes autodiscover provider can discover different kinds of resources, depending on the `resource` setting. When `resource: service` is used, it will collect events from services, and then I think it is fine to use `orchestrator.resource.type: service` there, but here it is discovering pods and containers, so I think that it is ok to use `pod`.